bootstrap/updater.go

```diff
@@ -1,3 +1,17 @@
+// Package bootstrap checks for the bootstrap/fallback data update from the
+// spacemesh administrator. This is intended as a short-term solution at the
```

Comment:
What is the spacemesh administrator?

Reply:
us. for genesis launch spacemesh will maintain a url that provides such network-wide update data.